README.md

```diff
@@ -3,3 +3,196 @@
 **This is not an official Google product.**
 
 Privacy data mapping and related plans management.
+
+## Set Up
```

Comment:
Have architecture and end user workflow before the set up.

Reply:
I am working on the architecture, and will add that later in a different PR.

Reply:
Discussed offline. I will use Qinhang's architecture graph in this PR to make it easier to review. Thanks to @sailorlqh for drawing the archiecture graph